flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/ResourceManager.java

```diff
@@ -401,74 +354,37 @@ public RegistrationResponse apply(RegistrationResponse registrationResponse, Thr
 	 * @return The response by the ResourceManager.
 	 */
 	@RpcMethod
-	public Future<RegistrationResponse> registerTaskExecutor(
```

Comment:
Unintended renaming? This method is annotated with `@RcpMethod`, we need to change `ResourceManagerGateway` as well. Although this begs the question why we expose an _internal_ method in the first place.